nemo/deploy/deploy_pytriton.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+try:
+    from pytriton.model_config import ModelConfig
+    from pytriton.triton import Triton, TritonConfig
+except:
+    pass
+from .deploy_base import DeployBase
+
+
+class DeployPyTriton(DeployBase):
```

Comment:
The classname has insufficient context. Deploy Pytriton with which model? We are planning to deploy pytriton for streaming asr and tts too as new tools, so please call this class `DeployPytritonLLM` or something that denotes what it is deploying.

Reply:
There is no need for a specific model. The goal is to support the deployment of any model. You can either pass a nemo ckpt using this param `checkpoint_path` or in memory model using this `model` param. The in memory model support will be added fully later though.

Reply:
Hmm, so you're saying the same code for streaming will apply to both LLM and ASR? Then no need to act on this comment, but let's keep it un resolved so ASR team can look into it later